TLM/TLM/Patcher.cs

```diff
@@ -41,19 +55,40 @@ public static class Patcher {
 
             if (fail) {
                 Log.Info("patcher failed");
-                UIView.library
-                        .ShowModal<ExceptionPanel>("ExceptionPanel")
-                        .SetMessage(
-                        "TM:PE failed to load",
-                        "Traffic Manager: President Edition failed to load. You can " +
-                        "continue playing but it's NOT recommended. Traffic Manager will " +
-                        "not work as expected.",
-                        true);
+                Shortcuts.ShowErrorDialog(
+                    "TM:PE failed to load",
+                    "Traffic Manager: President Edition failed to load. You can " +
+                    "continue playing but it's NOT recommended. Traffic Manager will " +
+                    "not work as expected.");
             } else {
                 Log.Info("TMPE patches installed successfully");
             }
         }
 
+        /// <summary>
+        /// applies all attribute diven harmony patches.
+        /// continues on error.
+        /// </summary>
+        /// <returns>false if exception happens, true otherwise</returns>
+        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
That method will never get inlined because of try...catch inside

Reply:
it does not hurt to emphasize :). even if not useful to the compiler it acts like doc so that programmer knows this function must not be inlined.